src/jvm/test/resources/expected_logs/atomic_references_name_two_fields_trace.txt

```diff
@@ -13,13 +13,13 @@ The following interleaving leads to the error:
 | ----------- |
 
 Detailed trace:
-| -------------------------------------------------------------------------------------------------------------------------------------------- |
-|                                                                   Thread 1                                                                   |
-| -------------------------------------------------------------------------------------------------------------------------------------------- |
-| operation()                                                                                                                                  |
-|   atomicReference1.READ: AtomicReference#1 at AtomicReferencesFromMultipleFieldsTest.operation(AtomicReferencesNamesTests.kt:97)             |
-|   atomicReference2.READ: AtomicReference#1 at AtomicReferencesFromMultipleFieldsTest.operation(AtomicReferencesNamesTests.kt:97)             |
-|   AtomicReference#1.get(): Node#1 at AtomicReferencesFromMultipleFieldsTest.operation(AtomicReferencesNamesTests.kt:97)                      |
-|   AtomicReference#1.compareAndSet(Node#1,Node#2): true at AtomicReferencesFromMultipleFieldsTest.operation(AtomicReferencesNamesTests.kt:97) |
-|   result: void                                                                                                                               |
-| -------------------------------------------------------------------------------------------------------------------------------------------- |
+| --------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                   Thread 1                                                                    |
+| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| operation()                                                                                                                                   |
+|   atomicReference1 ⟶ AtomicReference#1 at AtomicReferencesFromMultipleFieldsTest.operation(AtomicReferencesNamesTests.kt:97)                  |
```

Comment:
Please update the outputs

Reply:
they are updated

it is just on github both arrow types look identical, I guess